sources/packages/backend/apps/load-test-gateway/src/load-test-exception.filter.ts

```diff
@@ -0,0 +1,23 @@
+import { Catch, ArgumentsHost } from "@nestjs/common";
+import { BaseExceptionFilter } from "@nestjs/core";
+import { Request } from "express";
+import { LoggerService, InjectLogger } from "@sims/utilities/logger";
+
+@Catch()
+export class LoadTestAllExceptionsFilter extends BaseExceptionFilter {
+  catch(exception: unknown, host: ArgumentsHost) {
+    const ctx = host.switchToHttp();
+    const request: Request = ctx.getRequest();
+
+    // Logging Additional info
+    this.logger.error("Unhandled exception");
+    this.logger.error(`Request path [${request.path}]`);
+    this.logger.error(`${JSON.stringify(exception)}`);
```

Comment:
There is an issue with the way that we do the `JSON.stringify` that would prevent the log to be actually written. The  `parseJSONError` has the fix. I will create a tech debit to this be fixed in all the places.